packages/design-system-scripts/cli.js

```diff
@@ -20,24 +26,60 @@ yargs
     },
     handler: async argv => {
       const { build } = require('./gulp/build');
-      await build(argv.sourcePackageDir);
+      await build(argv.sourcePackageDir, { ...argv });
+    }
+  })
+  .command({
+    command: 'build-docs <sourcePackageDir> <docsPackageDirs..>',
```

Comment:
@pwolfert In what case would there be multiple `docsPackageDirs`? I thought these were the two use cases:
`build-docs packages/design-system packages/design-system-docs`
and
`build-docs ./src ./docs` (in the case of mgov ds)

Reply:
The second use case is actually
```
yarn cmsds build-docs . node_modules/@cmsgov/design-system-docs docs
```

Reply:
That's in order to inherit all the documentation from the core documentation package

Reply:
An alternative would be to read the package.json in the specified docs directory and to do the same thing we're doing for the assets in the source package, where if it's not the core one, we include the core one.

Reply:
https://github.com/CMSgov/design-system/pull/659/files#diff-b902b49a887f2e3a4fd421bf1e450047R46

Reply:
yeah i think maybe we could do that!